apps/api/src/app/events/dtos/trigger-event-response.dto.ts

```diff
@@ -11,7 +12,7 @@ export class TriggerEventResponseDto {
 
   @ApiProperty({
     description: 'Status for trigger',
-    enum: ['processed', 'trigger_not_active', 'subscriber_id_missing', 'error'],
+    enum: TriggerEventStatusEnum,
   })
   @IsString()
   @IsDefined()
```

Comment:
should we also update the `status: string;` to `status: TriggerEventStatusEnum;` and remove the `@IsString()`?

Reply:
Good catch!